js/components/report/data-fetch-error.js

```diff
@@ -28,7 +28,18 @@ export default class DataFetchError extends PureComponent {
     );
   }
 
+  renderGenericMessage(error) {
+    return (
+      <div>
+        <div>{error}</div>
+      </div>
+    );
+  }
+
   renderError(error) {
+    if (error.body) {
+      return this.renderGenericMessage(error.body);
```

Comment:
This is also untested, but I suspect testing this will be harder. So it'd be lower priority than the other untested lines.